storage/connect/domdoc.cpp

```diff
@@ -642,7 +642,7 @@ bool DOMNODELIST::DropItem(PGLOBAL g, int n)
 	if (Listp == NULL || Listp->length < n)
 		return true;
 
-//Listp->item[n] = NULL;  La propri�t� n'a pas de m�thode 'set'
+//Listp->item[n] = NULL;  La propriete n'a pas de methode 'set'
```

Comment:
In my opinion, we should be *translating* all of these non-English comments, so that all developers can read them. 

(**This one** can probably just be deleted since it's a long-commented line.)